ynr/apps/elections/uk/templates/candidates/person-view.html

```diff
@@ -142,7 +142,15 @@ <h2>Candidacies:</h2>
         {% else %}
            Contesting the {{ election_data.name }} ({{ election_data.election_date }})
         {% endif %}</dt>
+       
       <dd>{{ person|post_in_election:election_data }}</dd>
+      {% for membership in person.memberships.all %}
+      {% if membership.ballot.election == election_data %}
+          {% if membership.deselected %}
+            <dd>This candidate has been deselected by their party, but will remain on the ballot paper. <a href="{{ membership.deselected_source }}">Learn more<a>.</dd>
```

Comment:
Not a blocker, but it might be nice if this opened in a new tab. May just be personal preference 😅

Reply:
fancy, will add. good idea :)